README.md

```diff
@@ -85,7 +85,47 @@ echo "{
 ```
 
 ### Connecting to a local graph store
-As mentioned in the introduction, it is possible to connect `graph-notebook` to a graph database running on your local machine. An example being Gremlin Server. There are additional instructions regarding the use of local servers in the `additional-databases` folder.
+As mentioned in the introduction, it is possible to connect [`graph-notebook`](src/graph_notebook) to a graph database running on your local machine, an example being Gremlin Server. There are additional instructions regarding the use of local servers in the [`additional-databases`](additional-databases) folder.
```

Comment:
I would like to keep our primary README smaller, can we move this to a directory like we did for [gremlin-server](https://github.com/aws/graph-notebook/tree/main/additional-databases/gremlin-server)?